### PR TITLE
Add finalize target that copy locale folder and ca-bundle.crt 

### PR DESCRIPTION
**Pull request by talregev**
Add finalize target that copy locale folder and ca-bundle.crt  to output project.

### CODE REVIEW

**Review by AenBleidd — CHANGES_REQUESTED** [review]
I told you that I'll tell you how to do this properly ;)
This could be done in a more easy way. I'll write you in Discord later today with instructions what to do

File: win_build/boinc_finalize.targets [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Target Name="AfterBuild">
```
Comment (AenBleidd) [inline win_build/boinc_finalize.targets:2]:
```suggestion
	<Target Name="CopyCertificateAndLocaleFiles" DependsOnTargets="Build" AfterTargets="Build">
```

File: win_build/boinc_finalize.targets [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+		<ItemGroup>
+			<LOCALE Include="..\locale\**\*.*" />
+			<CERT Include="..\curl\ca-bundle.crt" />
+		</ItemGroup>
+		<Copy SourceFiles="@(LOCALE)" DestinationFolder="$(OutDir)\locale\%(RecursiveDir)" SkipUnchangedFiles="true" />
+		<Copy SourceFiles="@(CERT)"   DestinationFolder="$(OutDir)"                        SkipUnchangedFiles="true" />
+	</Target>
+</Project>
```
Comment (AenBleidd) [inline win_build/boinc_finalize.targets:10]: Please add new line

**Review by AenBleidd — CHANGES_REQUESTED** [review]

File: win_build/boinc_finalize.targets (hunk 2 above)
Comment (talregev) [reply win_build/boinc_finalize.targets:10]: I will. All other projects and target doesn't have new line.

Comment (AenBleidd) [reply win_build/boinc_finalize.targets]: I know but anyway for new files it's better to add it :)

**Comment by AenBleidd** [comment]
image

Next files should not be copied:
- Makefile.am
- update_templates.sh
- update_translations.sh

'templates' should not be copied too.

Also only *.mo files should be copied to language folders .

**Review by AenBleidd — APPROVED** [review]
LGTM

**Pull request merged**